src/Microsoft.OpenApi/Writers/OpenApiWriterBase.cs

```diff
@@ -4,9 +4,16 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Text.Json;
+using Json.Schema;
+using Json.Schema.OpenApi;
+using Microsoft.OpenApi.Any;
 using Microsoft.OpenApi.Exceptions;
+using Microsoft.OpenApi.Extensions;
 using Microsoft.OpenApi.Models;
 using Microsoft.OpenApi.Properties;
+using Microsoft.OpenApi.Services;
+using YamlDotNet.Serialization.ObjectGraphVisitors;
```

Comment:
We can't take this dependency. We are going to have to write the serialization of JsonSchema manually.